packages/express/src/accountsExpress.js

```diff
@@ -22,6 +22,27 @@ const accountsExpress = (AccountsServer) => {
   ? res.status(500).jsonp(err.serialize().message)
   : res.status(500).jsonp({ message: err.message }));
 
+  router.use(async (req, res, next) => {
+    const accessToken = get(req.headers, 'accounts-access-token', undefined) || get(req.body, 'accessToken', undefined);
+    if (!isEmpty(accessToken)) {
+      try {
+        const user = await AccountsServer.resumeSession(accessToken);
+        // eslint-disable-next-line no-param-reassign
+        AccountsServer.userId = () => user.id;
```

Comment:
I can't really think of a use case where I would use the accounts.userid function as opposed to just referencing the req.body, considering this just gets set anytime this route is hit (so really acts like a reference to the last logged in user -- is that the intent?)